Include/dictobject.h

```diff
@@ -57,6 +57,9 @@ PyAPI_FUNC(int) PyDict_MergeFromSeq2(PyObject *d,
 PyAPI_FUNC(PyObject *) PyDict_GetItemString(PyObject *dp, const char *key);
 PyAPI_FUNC(int) PyDict_SetItemString(PyObject *dp, const char *key, PyObject *item);
 PyAPI_FUNC(int) PyDict_DelItemString(PyObject *dp, const char *key);
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x03030000
```

Comment:
```suggestion
#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030A0000
```

Unfortunately, we can't backport this change, so we need to set the minimum version to 3.10.